konduit-serving-api/src/main/java/ai/konduit/serving/pipeline/step/ModelStep.java

```diff
@@ -66,7 +66,7 @@ public ModelStep(ModelConfig modelConfig) {
                 DataFormat.ND4J,
                 DataFormat.JSON,
                 DataFormat.IMAGE,
-                DataFormat.IMAGE
+//                DataFormat.IMAGE
```

Comment:
any reason for commenting this?

Reply:
Just because it was duplicated, no relation to the rest of the changes.

Reply:
Ah alright. Might as well just remove it then